Src/Extern/openPMD-api/AMReX_PlotFileUtilOPENPMD.H

```diff
@@ -0,0 +1,433 @@
+#ifndef AMREX_PLOTFILE_OPENPMD_API_H
+#define AMREX_PLOTFILE_OPENPMD_API_H
+
+#include <AMReX_Config.H>
+
+#include <AMReX_Geometry.H>
+#include <AMReX_MultiFab.H>
+#include <AMReX_PlotFileDataImpl.H>
+
+#ifdef AMREX_PARTICLES
+#include <AMReX_ParIter.H>
+#include <AMReX_Particles.H>
+#endif
+
+#ifdef AMREX_USE_OPENPMD_API
+#include <openPMD/openPMD.hpp>
+#endif
+
+#include <string>
+#include <memory>
+
+namespace amrex::openpmd_api
+{
+    namespace helper
+    {
+      inline void
+      setOpenPMDUnit ( openPMD::Mesh mesh, std::string const& field_name )
```

Comment:
This might be too specific for WarpX.

Reply:
OK I. can remove this and let apps handle it.